pkg/remote/github/github_team_details_fetcher.go

```diff
@@ -0,0 +1,35 @@
+package github
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/github"
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+)
+
+type GithubTeamDetailsFetcher struct {
+	reader       terraform.ResourceReader
+	deserializer *resource.Deserializer
+}
+
+func NewGithubTeamDetailsFetcher(provider terraform.ResourceReader, deserializer *resource.Deserializer) *GithubTeamDetailsFetcher {
+	return &GithubTeamDetailsFetcher{
+		reader:       provider,
+		deserializer: deserializer,
+	}
+}
+
+func (g *GithubTeamDetailsFetcher) ReadDetails(res resource.Resource) (resource.Resource, error) {
+	ctyVal, err := g.reader.ReadResource(terraform.ReadResourceArgs{
+		Ty: github.GithubTeamResourceType,
+		ID: res.TerraformId(),
+	})
+	if err != nil {
+		return nil, err
+	}
+	deserializedRes, err := g.deserializer.DeserializeOne(github.GithubTeamResourceType, *ctyVal)
+	if err != nil {
+		return nil, err
+	}
+
+	return deserializedRes, nil
+}
```

Comment:
Can't we use the generic details fetcher instead ?

Reply:
yep I'll update it didn't exists when I wrote it ^^